bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConstants.java

```diff
@@ -76,9 +76,9 @@ private static String getVersion() {
 
       Properties properties = new Properties();
       properties.load(stream);
-      String value = properties.getProperty("bigtable.hbase.version");
+      String value = properties.getProperty("bigtable.hbase.user_agent");
       if (value == null) {
-        LOG.error("bigtable.hbase.version not found in bigtable-hbase.properties.");
+        LOG.error("bigtable.hbase.user_agent not found in bigtable-hbase.properties.");
       } else if (value.startsWith("$")){
         LOG.info("project.version token is not replaced.");
```

Comment:
update log to mention ${project.artifactId}-${project.version} or just "Tokens are not replaced".